packages/swingset-runner/src/main.js

```diff
@@ -98,6 +99,46 @@ function fail(message, printUsage) {
   process.exit(1);
 }
 
+function generateSwingsetBenchmarkConfig(baseConfig, basedir, bootstrapArgv) {
```

Comment:
The explanation in the commit message is good; please add a version of it as a JSDoc comment on this function.